### PR TITLE
visualization of pie-chart widget

### PR DESCRIPTION
**Pull request by danielsinai**
# Description

Added explaination for how to create pie chart visualization

## Added docs pages

- Widgets (`docs/platform-overview/widgets/pie-chart.md`)

## Updated docs pages

Please also include the path for the updated docs

- Widgets (`docs/platform-overview/widgets/widgets.md`)

### CODE REVIEW

File: docs/platform-overview/widgets/pie-chart.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+sidebar_position: 5
+---
+
+# Pie Chart
+
+You can create a pie chart illustrating your catalog data divided by a specific category inside a specfic entity page [**Specific Entity Page**](../port-components/page.md#entity-page).
+
+## Creating visualization
+
+- On the top right of the page click on "Add Visualization" button
```
Comment (MPTG94) [inline docs/platform-overview/widgets/pie-chart.md:11]:
```suggestion
- On the top right of the page click on `Add Visualization` button:
```

File: docs/platform-overview/widgets/pie-chart.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+- Select Pie Chart
+
+![Dropdown](../../../static/img/platform-overview/widgets/AddPieChartVisualization.png)
+
+- Fill out the prompted form and click save
```
Comment (MPTG94) [inline docs/platform-overview/widgets/pie-chart.md:16]:
```suggestion
- Fill out the form and click save:
```

File: docs/platform-overview/widgets/pie-chart.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+![Dropdown](../../../static/img/platform-overview/widgets/AddPieChartForm.png)
+
+## Editing visualization
+
+- On the top right of the pie chart widget click on the three dots icon
+- Select your desired action (delete/edit)
```
Comment (MPTG94) [inline docs/platform-overview/widgets/pie-chart.md:23]:
```suggestion
- Select your desired action (edit/delete).
```

File: docs/platform-overview/widgets/pie-chart.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+![Dropdown](../../../static/img/platform-overview/widgets/AddPieChartVisualization.png)
+
+- Fill out the prompted form and click save
+
+![Dropdown](../../../static/img/platform-overview/widgets/AddPieChartForm.png)
+
+## Editing visualization
+
+- On the top right of the pie chart widget click on the three dots icon
```
Comment (MPTG94) [inline docs/platform-overview/widgets/pie-chart.md:22]:
```suggestion
- On the top right of the pie chart widget click on the three dots icon;
```

File: docs/platform-overview/widgets/pie-chart.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## Visualization properties
+
+| Field                   | Type     | Description                                                                                                            | Default | Required |
+| ----------------------- | -------- | ---------------------------------------------------------------------------------------------------------------------- | ------- | -------- |
+| `Title`                 | `String` | The title of the visualization                                                                                         | `null`  | `true`   |
+| `Icon`                  | `String` | The icon of the visualization                                                                                          | `null`  | `false`  |
+| `Description`           | `String` | A short description to describe the visualization                                                                      | `null`  | `false`  |
+| `Blueprint`             | `String` | The chart will display data about entities of the chosen blueprint that are related to this ServiceInEnv               | `null`  | `true`   |
```
Comment (MPTG94) [inline docs/platform-overview/widgets/pie-chart.md:34]:
```suggestion
| `Blueprint`             | `String` | The Blueprint to visualize the data of its Related Entities               | `null`  | `true`   |
```

File: docs/platform-overview/widgets/pie-chart.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+| `Breakdown by property` | `String` | Group your chart by a specific property                                                                                | `null`  | `true`   |
+| `Filters`               | `Array`  | Filters to include or exclue specific data [Read more](https://docs.getport.io/tutorials/search-in-port/#search-rules) | []      | `false`  |
```
Comment (MPTG94) [inline docs/platform-overview/widgets/pie-chart.md:36]:
```suggestion
| `Filters`               | `Array`  | Filters to include or exclue specific data based on Port's [Search Rules](https://docs.getport.io/tutorials/search-in-port/#search-rules) | []      | `false`  |
```

**Review by MPTG94 — APPROVED** [review]
LGTM

File: docs/platform-overview/widgets/pie-chart.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 5
+---
+
+# Pie Chart
+
+You can create a pie chart illustrating your catalog data divided by a specific category inside a specfic entity page [**Specific Entity Page**](../port-components/page.md#entity-page).
```
Comment (EllaFurman) [inline docs/platform-overview/widgets/pie-chart.md:7]:
```suggestion
# Pie chart

You can create a pie chart illustrating your software catalog divided by category inside a specific entity page [**Specific Entity Page**](../port-components/page.md#entity-page).
```

Comment (EllaFurman) [reply docs/platform-overview/widgets/pie-chart.md:7]: What do you mean by: "inside a specific entity page?"
How is it in an entity profile if it's a pie chart of the whole software catalog?

File: docs/platform-overview/widgets/pie-chart.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+## Creating visualization
+
+- On the top right of the page click on `Add Visualization` button:
+- Select Pie Chart
+
+![Dropdown](../../../static/img/platform-overview/widgets/AddPieChartVisualization.png)
+
+- Fill out the form and click save:
+
+![Dropdown](../../../static/img/platform-overview/widgets/AddPieChartForm.png)
```
Comment (EllaFurman) [inline docs/platform-overview/widgets/pie-chart.md:18]:
```suggestion
- On the top right of the page click on `Add Visualization` button;
- Select Pie Chart;

![Dropdown](../../../static/img/platform-overview/widgets/AddPieChartVisualization.png)

- Fill out the form and click save.

![Dropdown](../../../static/img/platform-overview/widgets/AddPieChartForm.png)
```

**Review by EllaFurman — CHANGES_REQUESTED** [review]

File: docs/platform-overview/widgets/pie-chart.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
+---
+sidebar_position: 5
+---
+
+# Pie chart
+
+You can create a pie chart illustrating data from Entities in your software catalog divided by categories and Entity properties inside a specific entity page [**Specific Entity Page**](../port-components/page.md#entity-page).
```
Comment (MPTG94) [inline docs/platform-overview/widgets/pie-chart.md:7]:
```suggestion
You can create a pie chart illustrating data from Entities in your software catalog divided by categories and Entity properties inside a Specific Entity Page [**Specific Entity Page**](../port-components/page.md#entity-page).
```

**Review by MPTG94 — APPROVED** [review]
Add small fix and merge

**Pull request merged**